kinto/core/resource/__init__.py

```diff
@@ -1087,7 +1087,9 @@ def _build_pagination_token(self, sorting, last_record, offset):
         }
 
         for field, _ in sorting:
-            token['last_record'][field] = last_record[field]
+            last_value = find_nested_value(last_record, field)
+            if last_value is not None:
+                token['last_record'][field] = last_value
```

Comment:
If `last_value` is `None` it's most probably because an erroneous sorting field key has been provided. Should we error is such a case, or just ignore it as I did?  We could possibly raise a `400` error properly describing the issue to the consumer, which sounds rather good to me. I can work on that if you want.